static/app/components/u2f/u2finterface.tsx

```diff
@@ -111,18 +126,49 @@ class U2fInterface extends React.Component<Props, State> {
       });
   }
 
+  webAuthnSignIn(authRequest) {
```

Comment:
is `authRequest` singular or plural? below its called implying its plural, but here its defined as singular.